api/api.py

```diff
@@ -40,6 +77,10 @@ def get_all_earthquakes() -> list[dict]:
     LEFT JOIN networks AS n USING(network_id)
     LEFT JOIN alerts AS a USING(alert_id)
     """
+    if any(value is not None for value in earthquake_filters.values()):
+        query_filter_commands = get_filter_queries(earthquake_filters)
+        search_query += " AND ".join(query_filter_commands)
+
     with get_cursor(conn) as cur:
         cur.execute(f"{search_query};")
```

Comment:
does this need to be an f string?

Reply:
I did this to add the apostrophe at the end because I thought it was required.
I went back and tested it and it looks like that is not the case. I will remove it.